code/processes/dqe.q

```diff
@@ -7,11 +7,46 @@ init:{
 
 gethandles:{exec procname,proctype,w from .servers.SERVERS where (procname in x) | (proctype in x)};
 
-tableexists:{[tabs]                                                                                             /- function to check for table, param is table name as a symbol
+tableexists:{[tab]                                                                                              /- function to check for table, param is table name as a symbol
   result:();
-  $[1=all a:tabs in tables[];result:(1b;"all tables exist");result:(0b;raze ("," sv string tabs where not a)," missing from process")]
+  .lg.o[`tabexist;"checking if ", (string tab), " table exists"];
```

Comment:
We should look to have the log on the client side identify that the log message is from dqe so to avoid debugging issues client side doesn't cause confusion.